### PR TITLE
DDF-1354: Added logging to track down integration test failures

### PR DESCRIPTION
**Pull request by lessarderic**
Added extra logging statements to WebSSOFilter and all the handlers to figure out if they all get registered before the first doFilter() call.

Also, increased integration test timeouts and added code to log the state of all the inactive bundles when the test setup fails to help with debugging.

@stustison, @andrewkfiedler please review.

Let me know if you're ok with these changes and if so I'll have @shaundmorris hero it.

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/101)

### CODE REVIEW

**Comment by andrew-fiedler** [comment]
:+1: Looks good!

---

Review status: 0 of 4 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

<sup>**[distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java, line 350 \[r1\]](https://reviewable.io:443/reviews/codice/ddf/101#-Jx0KfJCv_5yTBl1vKQk)** ([raw file](https://github.com/codice/ddf/blob/fe29ca32a7616f523eee08873e093f39d47e7c90/distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java#L350)):</sup>
Might be useful to provide information on which service failed to update.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/101)

**Comment by lessarderic** [comment]
<sup>**[distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java, line 350 \[r1\]](https://reviewable.io:443/reviews/codice/ddf/101#-Jx0KfJCv_5yTBl1vKQk)** ([raw file](https://github.com/codice/ddf/blob/fe29ca32a7616f523eee08873e093f39d47e7c90/distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java#L350)):</sup>
Added sourceConfig.getPid() to the log message. 

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/101)

**Comment by lessarderic** [comment]
@stustison, @shaundmorris, can you take a quick look so I can merge this in before the end of the sprint? @shaundmorris, can you also hero this PR?

File: platform/security/filter/security-filter-web-sso/src/main/java/org/codice/ddf/security/filter/websso/WebSSOFilter.java [hunk 1]
```diff
@@ -138,10 +149,16 @@ private void handleRequest(HttpServletRequest httpRequest, HttpServletResponse h
             FilterChain filterChain, List<AuthenticationHandler> handlers)
             throws IOException, ServletException {
 
-        // First pass, see if anyone can come up with proper security token from
-        // the git-go
+        if (handlers.size() == 0) {
+            LOGGER.warn("Handlers not ready. Returning status code 503, Service Unavailable.");
```
Comment (shaundmorris) [inline platform/security/filter/security-filter-web-sso/src/main/java/org/codice/ddf/security/filter/websso/WebSSOFilter.java]: org.codice.ddf.security.filter.websso.WebSSOFilterTest.testDoFilterForbidden fails here as it is still expecting a 400 code.

**Comment by lessarderic** [comment]
Not sure why that didn't get caught before but the code coverage has gone below the thresholds so I'll have to add a few unit tests to make the build pass again.

**Comment by shaundmorris** [comment]
Ah yes, I see that now as well

**Comment by lessarderic** [comment]
Ok, added some unit tests to increase the code coverage. @shaundmorris, you're currently the hero on this one but if you don't have time to take care of it let me know and I'll ask someone else.

**Comment by shaundmorris** [comment]
@lessarderic do you have a few minutes to check out the merge conflict?

**Comment by lessarderic** [comment]
@shaundmorris: Yes, looking into it.

**Pull request merged**